### PR TITLE
Change limit and add validation for product media alt field

### PR DESCRIPTION
**Pull request by jakubkuc**
I want to merge this change because it is a port of #15094  + merge migration

### CODE REVIEW

**Review by IKarbowiak — APPROVED** [review]

File: saleor/graphql/product/mutations/product/product_media_create.py [hunk 1]
```diff
@@ -80,6 +81,16 @@ def validate_input(cls, data):
                 }
             )
 
+        if alt and len(alt) > 250:
```
Comment (Air-t) [inline saleor/graphql/product/mutations/product/product_media_create.py:84]: why `250` and not a `ALT_CHAR_LIMIT `

File: saleor/graphql/product/mutations/product/product_media_create.py [hunk 2]
```diff
@@ -80,6 +81,16 @@ def validate_input(cls, data):
...
+            raise ValidationError(
+                {
+                    "input": ValidationError(
+                        "Alt field exceeds the character limit of 250.",
```
Comment (Air-t) [inline saleor/graphql/product/mutations/product/product_media_create.py:88]: same here

File: saleor/graphql/product/mutations/product/product_media_update.py [hunk 3]
```diff
@@ -45,6 +47,15 @@ def perform_mutation(  # type: ignore[override]
...
         if alt is not None:
+            if len(alt) > 250:
+                raise ValidationError(
+                    {
+                        "input": ValidationError(
+                            "Alt field exceeds the character limit of 250.",
+                            code=ProductErrorCode.INVALID.value,
+                        )
+                    }
+                )
```
Comment (Air-t) [inline saleor/graphql/product/mutations/product/product_media_update.py:58]: same here

File: saleor/product/models.py [hunk 4]
```diff
@@ -585,7 +585,7 @@ class ProductMedia(SortableModel, ModelWithMetadata):
         blank=True,
     )
     image = models.ImageField(upload_to="products", blank=True, null=True)
-    alt = models.CharField(max_length=128, blank=True)
+    alt = models.CharField(max_length=250, blank=True)
```
Comment (Air-t) [inline saleor/product/models.py:588]: why not 255 as a char field default limit is?

**Pull request merged**